node/src/cli.rs

```diff
@@ -87,6 +87,14 @@ pub enum Subcommand {
     #[structopt(name = "benchmark", about = "Benchmark runtime pallets.")]
     Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 
+    /// Try some command against runtime state.
+    #[cfg(feature = "try-runtime")]
+    TryRuntime(try_runtime_cli::TryRuntimeCmd),
+
+    /// Try some command against runtime state. Note: `try-runtime` feature must be enabled.
+    #[cfg(not(feature = "try-runtime"))]
+    TryRuntime,
```

Comment:
Why the two commands?

Reply:
This allows the command try-runtime to exist even when the feature is not enabled. In that case the command will just let the user know that they have to compile the node with the try-runtime feature enabled.

Reply:
Have you tried https://github.com/NodleCode/chain/pull/444#discussion_r767384666?

Reply:
I tested it now. This is what happens when the node is not built with the "try-runtime" and you try to call the "try-runtime" command:

> cargo run --release try-runtime                                               
    Finished release [optimized] target(s) in 0.59s
     Running `target/release/nodle-chain try-runtime`
Error: Input("TryRuntime wasn't enabled when building the node. You can enable it with `--features try-runtime`.")

Which seems what we should expect.